src/pages/books.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react';
+import BookCard from '../components/BookCard';
+import { IBook } from '../interfaces/books';
+
+const Books = () => {
+    const [books, setBooks] = useState<IBook[]>([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    const fetchBookData = async () => {
+        try {
+            const response = await fetch('/mockData/books.json');
+            const json = await response.json();
```

Comment:
Nit - it would be good to have a simple abstraction of this, maybe create a `fetchJSON` function that will do that instead of potentially doing this every time you need to fetch data
Nit - ideally, there should be a service for fetching data instead of hard coding URLs in components